src/scss/components/file/_hx-file-control.scss

```diff
@@ -0,0 +1,87 @@
+// ----- LAYOUT -----
+hx-file-control {
+  position: relative;
+
+  > * {
+    margin: 0;
+  }
+
+  > label {
+    display: inline-block;
+    padding: 0;
+  }
+
+  > input[type="file"] {
+    @include hxInput(visuallyHidden);
```

Comment:
Where is `hxInput` being loaded? Might need an explicit `@import`.

Reply:
@CITguy, as I understood our SCSS file structure, `src/scss/components/file/_hx-file-control.scss` is a partial.

Wouldn't `hxInput` be loaded through `src/scss/components/file/_index.scss`?

Reply:
added `@import "components/form/config"`